frontend/src/pages/RunListsRouter.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { RunStorageState } from 'src/apis/run';
+import MD2Tabs from 'src/atoms/MD2Tabs';
+import { commonCss, padding } from 'src/Css';
+import { classes } from 'typestyle';
+import RunList, { RunListProps } from './RunList';
+
+export enum RunListsGroupTab {
+  ACTIVE = 0,
+  ARCHIVE = 1,
+}
+
+export type RunListsRouterProps = RunListProps & {
+  storageState: RunStorageState;
+  onTabSwitch?: (tab: RunListsGroupTab) => void;
+};
+
+/**
+ * Contains two tab buttons which allows user to see Active or Archived runs list.
+ */
+class RunListsRouter extends React.PureComponent<RunListsRouterProps> {
+  protected _isMounted = true;
```

Comment:
Can I confirm, is this used anywhere in this component?

Reply:
Cleaned up.